RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __RecoLocalCalo_HGCalRecAlgos_RecHitTools_h__
+#define __RecoLocalCalo_HGCalRecAlgos_RecHitTools_h__
+
+#include <array>
+#include <cmath>
+
+class HGCalGeometry;
+class HGCalDDDConstants;
+class DetId;
+
+namespace edm {
+  class Event;
+  class EventSetup;
+}
+
+namespace hgcal {
+  class RecHitTools {    
```

Comment:
why is this code in Reco\* package at all?
The only dependence is on geometry and detid
and it operates only with objects that remain fixed during an IOV (forever, from MC perspective).
There is a getEvent method, but it is empty.

It appears to me that what's currently in the proposed RecHitTools, should instead be in an ESProduct and defined in the Geometry package or close to it.